iron-form.html

```diff
@@ -280,8 +280,16 @@
         if (!this._useValue(el) ||
             (el.hasAttribute('is') && json[el.name])) {
           continue;
+        } else if (el.tagName.toLowerCase() === 'select' && el.multiple) {
```

Comment:
we could also have `<input type="file" multiple>` which has a list of files `el.files.item(i)`. Should we already take care of it too in here?